tests/atest/rules/lengths/too_long_keyword/expected_output_severity_threshold.txt

```diff
@@ -1,2 +1,2 @@
-severity_threshold.robot:59:1 [W] 0501 Keyword 'Keyword' is too long (44/40)
-severity_threshold.robot:112:1 [E] 0501 Keyword 'Keyword2' is too long (51/40)
\ No newline at end of file
+severity_threshold.robot:15:1 [W] 0501 Keyword 'Keyword' is too long (44/40)
```

Comment:
So now we're pointing at the start of the keyword, not the end? (I'm fine with the change, just checking)

Reply:
Yes, because the `col` indicates where the underlining should start, and we want to underline the whole keyword, to make it visible for the user, not just the last character. 

In this case, the `end_col` is now what used to be the value of the `col`.

I think this:

<img width="122" alt="image" src="https://user-images.githubusercontent.com/7412964/226832297-8600b878-e122-44a5-948b-c123fa8b3929.png">

looks much better than this:

<img width="140" alt="image" src="https://user-images.githubusercontent.com/7412964/226832232-bfd9af74-2a5e-41ca-8d77-eb2b77ad7d52.png">